NoPortsInRangeException failure on Jenkins

I received the below exception in CI on Jenkins. It is unrelated to my work and both the test and the code appears untouched for months. Hence, I expect it to either have been flaky for some time or be affected by other changes.

**Stacktrace**
```
_______________________ test_find_available_port_forced ________________________

unused_tcp_port = 57880

    def test_find_available_port_forced(unused_tcp_port):
        custom_range = range(unused_tcp_port, unused_tcp_port)
        host, port, sock = port_handler.find_available_port(
>           custom_range=custom_range, reuse_addr=True
        )

/tmp/jenkins-scout-f_scout_ci/workspace/komodo_testing/ert-custom-script-pr@2/test-root/tests/ert_tests/shared/test_port_handler.py:23: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

custom_host = None, custom_range = range(57880, 57880), reuse_addr = True

    def find_available_port(
        custom_host: Optional[str] = None,
        custom_range: Optional[range] = None,
        reuse_addr: bool = False,
    ) -> Tuple[str, int, socket.socket]:
        current_host = custom_host if custom_host is not None else _get_ip_address()
        current_range = (
            custom_range if custom_range is not None else range(51820, 51840 + 1)
        )
        if current_range.start == current_range.stop:
            try:
                return (
                    current_host,
                    current_range.start,
                    _bind_socket(
                        host=current_host, port=current_range.start, reuse_addr=reuse_addr
                    ),
                )
            except PortAlreadyInUseException:
                pass
        else:
            attempts = 0
            while attempts < current_range.stop - current_range.start:
                try:
                    attempts += 1
                    num = random.randrange(current_range.start, current_range.stop)
                    return (
                        current_host,
                        num,
                        _bind_socket(host=current_host, port=num, reuse_addr=reuse_addr),
                    )
                except PortAlreadyInUseException:
                    continue
    
>       raise NoPortsInRangeException(f"No available ports in predefined {current_range}.")
E       ert_shared.port_handler.NoPortsInRangeException: No available ports in predefined range(57880, 57880).

/tmp/jenkins-scout-f_scout_ci/workspace/komodo_testing/ert-custom-script-pr@2/test-kenv/root/lib64/python3.6/site-packages/ert_shared/port_handler.py:52: NoPortsInRangeException
```

**Link to job**
https://ci.equinor.com/scout/job/komodo_testing/job/ert-custom-script-pr/6142/console

--- a/ert_shared/ensemble_evaluator/config.py
+++ b/ert_shared/ensemble_evaluator/config.py
@@ -143,9 +143,10 @@ class EvaluatorServerConfig:
         custom_port_range: typing.Optional[range] = None,
         use_token: bool = True,
         generate_cert: bool = True,
+        custom_host: typing.Optional[str] = None,
     ) -> None:
         self.host, self.port, self._socket_handle = port_handler.find_available_port(
-            custom_range=custom_port_range
+            custom_range=custom_port_range, custom_host=custom_host
         )
         self.protocol = "wss" if generate_cert else "ws"
         self.url = f"{self.protocol}://{self.host}:{self.port}"

--- a/tests/ert_tests/ensemble_evaluator/prefect/conftest.py
+++ b/tests/ert_tests/ensemble_evaluator/prefect/conftest.py
@@ -361,7 +361,9 @@ def function_ensemble_builder_factory(
 
 @pytest.fixture()
 def evaluator_config():
-    return EvaluatorServerConfig(custom_port_range=range(1024, 65535))
+    return EvaluatorServerConfig(
+        custom_port_range=range(1024, 65535), custom_host="127.0.0.1"
+    )
 
 
 class MockWSMonitor:

--- a/tests/ert_tests/ensemble_evaluator/test_ensemble_evaluator_config.py
+++ b/tests/ert_tests/ensemble_evaluator/test_ensemble_evaluator_config.py
@@ -24,7 +24,9 @@ def test_load_config(unused_tcp_port):
     assert not sock._closed
     sock.close()
 
-    ee_config = EvaluatorServerConfig(custom_port_range=range(1024, 65535))
+    ee_config = EvaluatorServerConfig(
+        custom_port_range=range(1024, 65535), custom_host="127.0.0.1"
+    )
     sock = ee_config.get_socket()
     assert sock is not None
     assert not sock._closed

--- a/tests/ert_tests/ensemble_evaluator/test_ensemble_evaluator_config.py
+++ b/tests/ert_tests/ensemble_evaluator/test_ensemble_evaluator_config.py
@@ -1,11 +1,12 @@
-from ert_shared import port_handler
 from ert_shared.ensemble_evaluator.config import EvaluatorServerConfig
 
 
 def test_load_config(unused_tcp_port):
     fixed_port = range(unused_tcp_port, unused_tcp_port)
-    serv_config = EvaluatorServerConfig(custom_port_range=fixed_port)
-    expected_host = port_handler._get_ip_address()
+    serv_config = EvaluatorServerConfig(
+        custom_port_range=fixed_port, custom_host="127.0.0.1"
+    )
+    expected_host = "127.0.0.1"
     expected_port = unused_tcp_port
     expected_url = f"wss://{expected_host}:{expected_port}"
     expected_client_uri = f"{expected_url}/client"

--- a/tests/ert_tests/ensemble_evaluator/test_ensemble_legacy.py
+++ b/tests/ert_tests/ensemble_evaluator/test_ensemble_legacy.py
@@ -14,7 +14,9 @@ def test_run_legacy_ensemble(tmpdir, make_ensemble_builder):
     custom_port_range = range(1024, 65535)
     with tmpdir.as_cwd():
         ensemble = make_ensemble_builder(tmpdir, num_reals, 2).build()
-        config = EvaluatorServerConfig(custom_port_range=custom_port_range)
+        config = EvaluatorServerConfig(
+            custom_port_range=custom_port_range, custom_host="127.0.0.1"
+        )
         evaluator = EnsembleEvaluator(ensemble, config, 0, ee_id="1")
         with evaluator.run() as monitor:
             for e in monitor.track():
@@ -36,7 +38,9 @@ def test_run_and_cancel_legacy_ensemble(tmpdir, make_ensemble_builder):
     custom_port_range = range(1024, 65535)
     with tmpdir.as_cwd():
         ensemble = make_ensemble_builder(tmpdir, num_reals, 2, job_sleep=5).build()
-        config = EvaluatorServerConfig(custom_port_range=custom_port_range)
+        config = EvaluatorServerConfig(
+            custom_port_range=custom_port_range, custom_host="127.0.0.1"
+        )
 
         evaluator = EnsembleEvaluator(ensemble, config, 0, ee_id="1")
 
@@ -56,7 +60,9 @@ def test_run_legacy_ensemble_exception(tmpdir, make_ensemble_builder):
     custom_port_range = range(1024, 65535)
     with tmpdir.as_cwd():
         ensemble = make_ensemble_builder(tmpdir, num_reals, 2).build()
-        config = EvaluatorServerConfig(custom_port_range=custom_port_range)
+        config = EvaluatorServerConfig(
+            custom_port_range=custom_port_range, custom_host="127.0.0.1"
+        )
         evaluator = EnsembleEvaluator(ensemble, config, 0, ee_id="1")
 
         with patch.object(ensemble, "get_active_reals", side_effect=RuntimeError()):

--- a/tests/ert_tests/ensemble_evaluator/test_sync_ws_duplexer.py
+++ b/tests/ert_tests/ensemble_evaluator/test_sync_ws_duplexer.py
@@ -141,7 +141,9 @@ def test_generator(unused_tcp_port, ws):
 
 def test_secure_echo(ws):
     custom_port_range = range(1024, 65535)
-    config = EvaluatorServerConfig(custom_port_range=custom_port_range)
+    config = EvaluatorServerConfig(
+        custom_port_range=custom_port_range, custom_host="127.0.0.1"
+    )
 
     async def handler(websocket, path):
         msg = await websocket.recv()

--- a/tests/ert_tests/shared/test_port_handler.py
+++ b/tests/ert_tests/shared/test_port_handler.py
@@ -492,7 +492,9 @@ def test_reuse_passive_live_linux_nok_ok(unused_tcp_port):
 
     # As long as the socket is kept alive this port can not be bound again...
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
-        port_handler.find_available_port(custom_range=custom_range)
+        port_handler.find_available_port(
+            custom_range=custom_range, custom_host="127.0.0.1"
+        )
 
     # ... but on Linux the port can be re-bound by setting this flag!
     # This does not seem safe in a multi-user/-process environment!

--- a/tests/ert_tests/shared/test_port_handler.py
+++ b/tests/ert_tests/shared/test_port_handler.py
@@ -7,7 +7,9 @@ from ert_shared import port_handler
 
 def test_find_available_port(unused_tcp_port):
     custom_range = range(unused_tcp_port, unused_tcp_port + 1)
-    host, port, sock = port_handler.find_available_port(custom_range=custom_range)
+    host, port, sock = port_handler.find_available_port(
+        custom_range=custom_range, custom_host="127.0.0.1"
+    )
     assert host is not None
     assert port is not None
     assert port in custom_range
@@ -17,7 +19,9 @@ def test_find_available_port(unused_tcp_port):
 
 def test_find_available_port_forced(unused_tcp_port):
     custom_range = range(unused_tcp_port, unused_tcp_port)
-    host, port, sock = port_handler.find_available_port(custom_range=custom_range)
+    host, port, sock = port_handler.find_available_port(
+        custom_range=custom_range, custom_host="127.0.0.1"
+    )
     assert port == unused_tcp_port
     assert sock is not None
     assert sock.fileno() != -1
@@ -27,7 +31,9 @@ def test_invalid_host_name():
     invalid_host = "invalid_host"
 
     with pytest.raises(port_handler.InvalidHostException) as exc_info:
-        port_handler.find_available_port(custom_host=invalid_host)
+        port_handler.find_available_port(
+            custom_host=invalid_host, custom_host="127.0.0.1"
+        )
 
     assert (
         f"Trying to bind socket with what looks like an invalid hostname ({invalid_host})"
@@ -49,7 +55,9 @@ def test_get_family():
 def test_gc_closes_socket(unused_tcp_port):
     custom_range = range(unused_tcp_port, unused_tcp_port + 1)
 
-    _, port, orig_sock = port_handler.find_available_port(custom_range=custom_range)
+    _, port, orig_sock = port_handler.find_available_port(
+        custom_range=custom_range, custom_host="127.0.0.1"
+    )
     assert port == unused_tcp_port
     assert orig_sock is not None
     assert orig_sock.fileno() != -1
@@ -64,7 +72,9 @@ def test_gc_closes_socket(unused_tcp_port):
 
     orig_sock = None
 
-    _, port, orig_sock = port_handler.find_available_port(custom_range=custom_range)
+    _, port, orig_sock = port_handler.find_available_port(
+        custom_range=custom_range, custom_host="127.0.0.1"
+    )
     assert port == unused_tcp_port
     assert orig_sock is not None
     assert orig_sock.fileno() != -1
@@ -166,7 +176,9 @@ def test_def_passive_live_nok_nok_close_ok_ok(unused_tcp_port):
     custom_range = range(unused_tcp_port, unused_tcp_port + 1)
 
     # Opening original socket with will_close_then_reopen_socket=False
-    _, port, orig_sock = port_handler.find_available_port(custom_range=custom_range)
+    _, port, orig_sock = port_handler.find_available_port(
+        custom_range=custom_range, custom_host="127.0.0.1"
+    )
     assert port == unused_tcp_port
     assert orig_sock is not None
     assert orig_sock.fileno() != -1
@@ -174,18 +186,24 @@ def test_def_passive_live_nok_nok_close_ok_ok(unused_tcp_port):
     # When the socket is kept open, this port can not be reused
     # with or without setting will_close_then_reopen_socket
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
-        port_handler.find_available_port(custom_range=custom_range)
+        port_handler.find_available_port(
+            custom_range=custom_range, custom_host="127.0.0.1"
+        )
 
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
         port_handler.find_available_port(
-            custom_range=custom_range, will_close_then_reopen_socket=True
+            custom_range=custom_range,
+            custom_host="127.0.0.1",
+            will_close_then_reopen_socket=True,
         )
 
     orig_sock.close()
 
     # When we close the socket without actually having used it, it is
     # immediately reusable with or without setting will_close_then_reopen_socket
-    _, port, sock = port_handler.find_available_port(custom_range=custom_range)
+    _, port, sock = port_handler.find_available_port(
+        custom_range=custom_range, custom_host="127.0.0.1"
+    )
     assert port == unused_tcp_port
     assert sock is not None
     assert sock.fileno() != -1
@@ -194,7 +212,9 @@ def test_def_passive_live_nok_nok_close_ok_ok(unused_tcp_port):
     sock.close()
 
     _, port, sock = port_handler.find_available_port(
-        custom_range=custom_range, will_close_then_reopen_socket=True
+        custom_range=custom_range,
+        custom_host="127.0.0.1",
+        will_close_then_reopen_socket=True,
     )
     assert port == unused_tcp_port
     assert sock is not None
@@ -215,7 +235,9 @@ def test_reuse_active_close_nok_ok(unused_tcp_port):
 
     # Note: Setting will_close_then_reopen_socket=True on original socket
     host, port, orig_sock = port_handler.find_available_port(
-        custom_range=custom_range, will_close_then_reopen_socket=True
+        custom_range=custom_range,
+        custom_host="127.0.0.1",
+        will_close_then_reopen_socket=True,
     )
     assert port == unused_tcp_port
     assert orig_sock is not None
@@ -227,11 +249,15 @@ def test_reuse_active_close_nok_ok(unused_tcp_port):
 
     # Using will_close_then_reopen_socket=False fails...
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
-        port_handler.find_available_port(custom_range=custom_range)
+        port_handler.find_available_port(
+            custom_range=custom_range, custom_host="127.0.0.1"
+        )
 
     # ... but using will_close_then_reopen_socket=True succeeds
     _, port, sock = port_handler.find_available_port(
-        custom_range=custom_range, will_close_then_reopen_socket=True
+        custom_range=custom_range,
+        custom_host="127.0.0.1",
+        will_close_then_reopen_socket=True,
     )
     assert port == unused_tcp_port
     assert sock is not None
@@ -251,7 +277,9 @@ def test_reuse_active_live_nok_nok(unused_tcp_port):
     custom_range = range(unused_tcp_port, unused_tcp_port + 1)
 
     host, port, orig_sock = port_handler.find_available_port(
-        custom_range=custom_range, will_close_then_reopen_socket=True
+        custom_range=custom_range,
+        custom_host="127.0.0.1",
+        will_close_then_reopen_socket=True,
     )
     assert port == unused_tcp_port
     assert orig_sock is not None
@@ -263,11 +291,15 @@ def test_reuse_active_live_nok_nok(unused_tcp_port):
     # Even with "will_close_then_reopen_socket"=True when obtaining original
     # socket, subsequent calls fails
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
-        port_handler.find_available_port(custom_range=custom_range)
+        port_handler.find_available_port(
+            custom_range=custom_range, custom_host="127.0.0.1"
+        )
 
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
         _, port, sock = port_handler.find_available_port(
-            custom_range=custom_range, will_close_then_reopen_socket=True
+            custom_range=custom_range,
+            custom_host="127.0.0.1",
+            will_close_then_reopen_socket=True,
         )
 
 
@@ -282,7 +314,9 @@ def test_def_active_live_nok_nok(unused_tcp_port):
     """
     custom_range = range(unused_tcp_port, unused_tcp_port + 1)
 
-    host, port, orig_sock = port_handler.find_available_port(custom_range=custom_range)
+    host, port, orig_sock = port_handler.find_available_port(
+        custom_range=custom_range, custom_host="127.0.0.1"
+    )
     assert port == unused_tcp_port
     assert orig_sock is not None
     assert orig_sock.fileno() != -1
@@ -292,12 +326,16 @@ def test_def_active_live_nok_nok(unused_tcp_port):
 
     # Immediately trying to bind to the same port fails...
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
-        host, port, sock = port_handler.find_available_port(custom_range=custom_range)
+        host, port, sock = port_handler.find_available_port(
+            custom_range=custom_range, custom_host="127.0.0.1"
+        )
 
     # ... also using will_close_then_reopen_socket=True
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
         host, port, sock = port_handler.find_available_port(
-            custom_range=custom_range, will_close_then_reopen_socket=True
+            custom_range=custom_range,
+            custom_host="127.0.0.1",
+            will_close_then_reopen_socket=True,
         )
 
 
@@ -316,7 +354,9 @@ def test_def_active_close_macos_nok_ok(unused_tcp_port):
     """
     custom_range = range(unused_tcp_port, unused_tcp_port + 1)
 
-    host, port, orig_sock = port_handler.find_available_port(custom_range=custom_range)
+    host, port, orig_sock = port_handler.find_available_port(
+        custom_range=custom_range, custom_host="127.0.0.1"
+    )
     assert port == unused_tcp_port
     assert orig_sock is not None
     assert orig_sock.fileno() != -1
@@ -327,12 +367,16 @@ def test_def_active_close_macos_nok_ok(unused_tcp_port):
 
     # Immediately trying to bind to the same port fails
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
-        _, _, sock = port_handler.find_available_port(custom_range=custom_range)
+        _, _, sock = port_handler.find_available_port(
+            custom_range=custom_range, custom_host="127.0.0.1"
+        )
 
     # On MacOS, setting will_close_then_reopen_socket=True in subsequent calls allows
     # to reuse the port
     _, port, sock = port_handler.find_available_port(
-        custom_range=custom_range, will_close_then_reopen_socket=True
+        custom_range=custom_range,
+        custom_host="127.0.0.1",
+        will_close_then_reopen_socket=True,
     )
     assert port == unused_tcp_port
     assert sock is not None
@@ -354,7 +398,9 @@ def test_def_active_close_linux_nok_nok(unused_tcp_port):
     """
     custom_range = range(unused_tcp_port, unused_tcp_port + 1)
 
-    host, port, orig_sock = port_handler.find_available_port(custom_range=custom_range)
+    host, port, orig_sock = port_handler.find_available_port(
+        custom_range=custom_range, custom_host="127.0.0.1"
+    )
     assert port == unused_tcp_port
     assert orig_sock is not None
     assert orig_sock.fileno() != -1
@@ -365,13 +411,17 @@ def test_def_active_close_linux_nok_nok(unused_tcp_port):
 
     # Immediately trying to bind to the same port fails
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
-        host, port, sock = port_handler.find_available_port(custom_range=custom_range)
+        host, port, sock = port_handler.find_available_port(
+            custom_range=custom_range, custom_host="127.0.0.1"
+        )
 
     # On Linux, setting will_close_then_reopen_socket=True in subsequent calls do
     # NOT allow reusing the port in this case
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
         host, port, sock = port_handler.find_available_port(
-            custom_range=custom_range, will_close_then_reopen_socket=True
+            custom_range=custom_range,
+            custom_host="127.0.0.1",
+            will_close_then_reopen_socket=True,
         )
 
 
@@ -390,7 +440,9 @@ def test_reuse_passive_live_macos_nok_nok(unused_tcp_port):
     custom_range = range(unused_tcp_port, unused_tcp_port + 1)
 
     _, port, orig_sock = port_handler.find_available_port(
-        custom_range=custom_range, will_close_then_reopen_socket=True
+        custom_range=custom_range,
+        custom_host="127.0.0.1",
+        will_close_then_reopen_socket=True,
     )
     assert port == unused_tcp_port
     assert orig_sock is not None
@@ -398,12 +450,16 @@ def test_reuse_passive_live_macos_nok_nok(unused_tcp_port):
 
     # As long as the socket is kept alive this port can not be bound again...
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
-        port_handler.find_available_port(custom_range=custom_range)
+        port_handler.find_available_port(
+            custom_range=custom_range, custom_host="127.0.0.1"
+        )
 
     # ... not even when setting will_close_then_reopen_socket=True
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
         port_handler.find_available_port(
-            custom_range=custom_range, will_close_then_reopen_socket=True
+            custom_range=custom_range,
+            custom_host="127.0.0.1",
+            will_close_then_reopen_socket=True,
         )
 
 
@@ -424,7 +480,9 @@ def test_reuse_passive_live_linux_nok_ok(unused_tcp_port):
 
     # Opening original socket with will_close_then_reopen_socket=True
     _, port, orig_sock = port_handler.find_available_port(
-        custom_range=custom_range, will_close_then_reopen_socket=True
+        custom_range=custom_range,
+        custom_host="127.0.0.1",
+        will_close_then_reopen_socket=True,
     )
     assert port == unused_tcp_port
     assert orig_sock is not None
@@ -437,7 +495,9 @@ def test_reuse_passive_live_linux_nok_ok(unused_tcp_port):
     # ... but on Linux the port can be re-bound by setting this flag!
     # This does not seem safe in a multi-user/-process environment!
     _, port, sock = port_handler.find_available_port(
-        custom_range=custom_range, will_close_then_reopen_socket=True
+        custom_range=custom_range,
+        custom_host="127.0.0.1",
+        will_close_then_reopen_socket=True,
     )
     assert port == unused_tcp_port
     assert sock is not None
@@ -456,7 +516,9 @@ def test_reuse_passive_close_ok_ok(unused_tcp_port):
     custom_range = range(unused_tcp_port, unused_tcp_port + 1)
 
     _, port, orig_sock = port_handler.find_available_port(
-        custom_range=custom_range, will_close_then_reopen_socket=True
+        custom_range=custom_range,
+        custom_host="127.0.0.1",
+        will_close_then_reopen_socket=True,
     )
     assert port == unused_tcp_port
     assert orig_sock is not None
@@ -466,7 +528,9 @@ def test_reuse_passive_close_ok_ok(unused_tcp_port):
 
     # When we close the socket without actually having used it, it is
     # immediately reusable with or without setting will_close_then_reopen_socket
-    _, port, sock = port_handler.find_available_port(custom_range=custom_range)
+    _, port, sock = port_handler.find_available_port(
+        custom_range=custom_range, custom_host="127.0.0.1"
+    )
     assert port == unused_tcp_port
     assert sock is not None
     assert sock.fileno() != -1
@@ -475,7 +539,9 @@ def test_reuse_passive_close_ok_ok(unused_tcp_port):
     sock.close()
 
     _, port, sock = port_handler.find_available_port(
-        custom_range=custom_range, will_close_then_reopen_socket=True
+        custom_range=custom_range,
+        custom_host="127.0.0.1",
+        will_close_then_reopen_socket=True,
     )
     assert port == unused_tcp_port
     assert sock is not None

--- a/tests/ert_tests/shared/test_port_handler.py
+++ b/tests/ert_tests/shared/test_port_handler.py
@@ -31,9 +31,7 @@ def test_invalid_host_name():
     invalid_host = "invalid_host"
 
     with pytest.raises(port_handler.InvalidHostException) as exc_info:
-        port_handler.find_available_port(
-            custom_host=invalid_host, custom_host="127.0.0.1"
-        )
+        port_handler.find_available_port(custom_host=invalid_host)
 
     assert (
         f"Trying to bind socket with what looks like an invalid hostname ({invalid_host})"
@@ -63,11 +61,15 @@ def test_gc_closes_socket(unused_tcp_port):
     assert orig_sock.fileno() != -1
 
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
-        port_handler.find_available_port(custom_range=custom_range)
+        port_handler.find_available_port(
+            custom_range=custom_range, custom_host="127.0.0.1"
+        )
 
     with pytest.raises(port_handler.NoPortsInRangeException) as exc_info:
         port_handler.find_available_port(
-            custom_range=custom_range, will_close_then_reopen_socket=True
+            custom_range=custom_range,
+            will_close_then_reopen_socket=True,
+            custom_host="127.0.0.1",
         )
 
     orig_sock = None

--- a/tests/ert_tests/status/test_evaluator_tracker.py
+++ b/tests/ert_tests/status/test_evaluator_tracker.py
@@ -317,7 +317,9 @@ def test_tracking_progress(
         brm = run_model()
     else:
         brm = run_model(None, None)
-    ee_config = EvaluatorServerConfig()
+    ee_config = EvaluatorServerConfig(
+        custom_port_range=range(1024, 65535), custom_host="127.0.0.1"
+    )
     with patch(
         "ert.ensemble_evaluator.tracker.evaluator_tracker.create_ee_monitor"
     ) as mock_ee:

--- a/tests/ert_tests/status/test_tracking_integration.py
+++ b/tests/ert_tests/status/test_tracking_integration.py
@@ -146,7 +146,7 @@ def test_tracking(
             model, argument = create_model(ert, parsed)
 
             evaluator_server_config = EvaluatorServerConfig(
-                custom_port_range=range(1024, 65535)
+                custom_port_range=range(1024, 65535), custom_host="127.0.0.1"
             )
 
             thread = threading.Thread(

